mmv1/products/vertexai/Schedule.yaml

```diff
@@ -0,0 +1,323 @@
+    exactly_one_of:
+      - request.0.create_pipeline_job_request
+    properties:
+      - !ruby/object:Api::Type::NestedObject
+        name: createPipelineJobRequest
+        description: |
+          Request for PipelineService.CreatePipelineJob.
+          CreatePipelineJobRequest.parent field is required
+          (format: projects/{project}/locations/{location}).
+        properties:
+          - !ruby/object:Api::Type::String
+            name: parent
+            description: Required. The resource name of the Location to create the PipelineJob in.
+              Format: projects/{project}/locations/{location}
+            required: true
+          - !ruby/object:Api::Type::NestedObject
+            name: pipelineJob
+            description: Required. The PipelineJob to create.
+            required: true
+            properties:
+              - !ruby/object:Api::Type::String
+                name: displayName
+                description: |
+                  The display name of the Pipeline. The name can be up to 128 characters long
+                  and can consist of any UTF-8 characters.
+              - !ruby/object:Api::Type::NestedObject
+                name: pipelineSpec
+                description: The spec of the pipeline.
+                required: true
+                # TODO: add pipelineSpec properties
+              - !ruby/object:Api::Type::KeyValuePairs
+                name: labels
+                description: |
+                  The labels with user-defined metadata to organize PipelineJob.
+                  Label keys and values can be no longer than 64 characters
+                  (Unicode codepoints), can only contain lowercase letters,
+                  numeric characters, underscores and dashes. International
+                  characters are allowed. See https://goo.gl/xmQnxf for more
+                  information and examples of labels.
+              - !ruby/object:Api::Type::NestedObject
+                name: runtimeConfig
+                description: Runtime config of the pipeline.
+                required: true
+                properties:
+                  # TODO parameters
+                  - !ruby/object:Api::Type::String
+                    name: gcsOutputDirectory
+                    description: |
+                      Required. A path in a Cloud Storage bucket, which will be treated as the root
+                      output directory of the pipeline. It is used by the system to generate the paths
+                      of output artifacts. The artifact paths are generated with a sub-path pattern
+                      {job_id}/{taskId}/{output_key} under the specified output directory.
+                      The service account specified in this pipeline must have the storage.objects.get
+                      and storage.objects.create permissions for this bucket.
+                    required: true
+                  # TODO parameterValues
```

Comment:
Not sure how to implement this type in magic modules
`map (key: string, value: value (Value format))`

https://cloud.google.com/vertex-ai/docs/reference/rest/v1beta1/projects.locations.pipelineJobs#PipelineJob.RuntimeConfig

Reply:
Does a key value pair work here?

Reply:
likewise here - I think key value pair only works for string values?